_beats/dev-tools/generate_notice.py

```diff
@@ -273,7 +282,12 @@ def detect_license_summary(content):
             if exclude in dirs:
                 dirs.remove(exclude)
 
+    overrides = {}  # revision overrides only for now
+    if args.beats_origin:
+        govendor = json.load(args.beats_origin)
+        overrides = {package['path']: package for package in govendor["package"]}
+
     print("Get the licenses available from {}".format(vendor_dirs))
-    create_notice(notice, args.beat, args.copyright, vendor_dirs, args.csvfile)
+    create_notice(notice, args.beat, args.copyright, vendor_dirs, args.csvfile, overrides=overrides)
```

Comment:
I think this could lead to inconsistencies if the same third party lib is also used as a direct lib, as the versions could differ (haven't tried it out though). As I don't think that this is the case for us atm I'm fine with merging this and discuss if we need further changes in a follow up.